src/Components/Form.js

```diff
@@ -44,13 +59,63 @@ class Form extends React.Component {
             placeholder="Age "
           />
         </div>
-        <Dropdown id="1" />
-        <Radio />
+        <div>
+          <label id="dropdown-label" htmlFor="dropdown">
+            Which option best describes your current role?
+          </label>
+          <select id="dropdown" name="dropdown">
+            <Option
+              value="Select current role"
+              disabled={true}
+              selected={true}
+            />
+            <Option value="Student" />
+            <Option value="Full Time Job" />
+            <Option value="Full Time learner" />
+            <Option value="Prefer not to say" />
+            <Option value="Other" />
+          </select>
+        </div>
+        <div>
+          <label htmlFor="recommendation">
+            Would you recommend freeCodeCamp to a friend?
+          </label>
+          <div id="recommendation">
+            <Radio
+              value="definitely"
+              label="Definitely"
+              name="recommendation"
+              checked={true}
+            />
+            <Radio value="maybe" label="Maybe" name="recommendation" />
+            <Radio value="not-sure" label="Not sure" name="recommendation" />
+          </div>
+        </div>
+        <div>
+          <label htmlFor="improved">
+            What would you like to see improved?(check all that apply){" "}
+          </label>
+          <div id="improved">
+            <CheckBox checkboxData={this.state.checkboxData} />
```

Comment:
Let's have this checkbox to represent only one checkbox in our system.
So the map function should be part of this component not the Checkbox component.